pkg/errortypes/supernode_errors.go

```diff
@@ -32,7 +32,7 @@ var (
 
 	// ErrUnknowError represents the error should not happen
 	// and the cause of that is unknown.
-	ErrUnknowError = DfError{codeUnknowError, "unknow error"}
+	ErrUnknowError = DfError{codeUnknowError, "unknown error"}
```

Comment:
Should it be ErrUnknownError?

Reply:
@yuxiaobo96  If you want to address the typo, please address all of them such as `ErrUnknowError`, `codeUnknowError`

Reply:
Ok,